engine/src/w32stack.cpp

```diff
@@ -603,8 +603,12 @@ void MCStack::constrain(intptr_t lp)
 
 	// MW-2007-07-27: In Windows 98 we need to clamp to 32767...
 	
-	mmptr -> ptMinTrackSize . x = minwidth + dx;
-	mmptr -> ptMinTrackSize . y = minheight + dy;
+	double *t_screenPixelScale;
+	uindex_t t_count;
+	MCResListScreenPixelScales(false, t_count, t_screenPixelScale);
+
+	mmptr -> ptMinTrackSize . x = minwidth * (*t_screenPixelScale) + dx;
```

Comment:
@livecodealex we get the correct display above on L580 so we don't need to do this. Just `&t_display -> pixel_scale` should cover us.